capstone-frontend/src/components/ListPage.jsx

```diff
@@ -79,49 +103,78 @@ class ListPage extends Component {
         <h1>Loading...</h1>
       </div>)
       :
-      (this.state.empty
+      ((this.state.gBooks.length === 0)
         ?
         (
           <div>
-            <h2 className="mt-3">{this.state.bookList.name}</h2>
-            <hr className="light-gray-border" />
+            <Row>
+              <Col>
+                <h2 className="mt-4 ml-2">{this.state.bookList.name}</h2>
+              </Col>
+              <Col className="text-right">
+                <SearchBookModal
+                  objectId={this.props.match.params.id}
+                  update={this.updateListPage}
+                  putURL="/api/booklist"
+                  type="booklist"
+                  btnStyle="btn btn-primary mt-4 mr-2" />
+              </Col>
+            </Row>
+            <hr className="light-gray-border mx-2 my-2" />
             <h3 className="text-center mt-4">Booklist has No Books</h3>
           </div>
         )
-        : (
+        :
+        (
           <div>
-            <h2 className="mt-3">{this.state.bookList.name}</h2>
-            <hr className="light-gray-border" />
+            <Row>
+              <Col>
+                <h2 className="mt-4 ml-2">{this.state.bookList.name}</h2>
+              </Col>
+              <Col className="text-right">
+                <SearchBookModal
+                  objectId={this.props.match.params.id}
+                  update={this.updateListPage}
+                  putURL="/api/booklist"
+                  type="booklist"
+                  btnStyle="btn btn-primary mt-4 mr-2" />
+              </Col>
+            </Row>
+            <hr className="light-gray-border mx-2 my-2" />
 
             <div>
               {
-                this.state.gBooks.map(gBook =>
-                  <Row className="text-center border m-5 bg-light light-gray-border" key={gBook.id + this.props.match.params.id} >
-                    <Col md={3} className="my-4 p-0 ">
-                      {/* TODO(#79): Redirect user to BookPage instead of playstore */}
-                      <a className="text-decoration-none text-body" href={gBook.canonicalVolumeLink}>
-                        <img className="img-responsive" src={gBook.thumbnailLink} alt={gBook.title} />
-                      </a>
-                    </Col>
-
-                    <Col className="my-4 p-0">
-                      <h2 className="mt-4"> {gBook.title} </h2>
-                      <StarRatings
-                        rating={gBook.avgRating}
-                        starDimension="40px"
-                        starSpacing="10px"
-                        starRatedColor="gold" />
+                (this.state.gBooks.length !== 0) &&
```

Comment:
Since you added the `(this.state.gBooks.length === 0)` check on line 106, do you still need to make this check here?

Reply:
No I don't, Thanks.